doc/interop-test-descriptions.md

```diff
@@ -1069,6 +1069,50 @@ included in that latency measurement (channel teardown semantics differ widely
 between languages). This latency measurement should also be the value that is
 logged and recorded in the latency histogram.
 
+
+### orca
+
+The client verifies that a custom LB policy, which is integrated with ORCA APIs, will receive both 
+per-query and out-of-band metric reports from the backend. 
+
+The client will register the custom LB policy named `test_backend_metrics_load_balancer`, which
+using ORCA APIs already installed a per-query report listener, and an out-of-band report 
+listener (configure load report interval to be 1s). The interop-testing client will run with a
+service config to select the load balancing config (using argument `--service_config_json`), so that 
+it effectively uses this newly registered custom LB policy. A load report reference can be passed 
+from the call to the LB policy through, e.g. CallOptions, to receive metric reports. 
+The reference contains the load report for the current RPC in the per-query case, or the latest report
+in the out-of-band case. This way, together with server behaviors we can verify the 
+expected metric reports are received.
+
+To perform the test, two new fields are added to the test unary call request data structure to 
+communicate with the server:
+1. A `TestLoadReport` type `orca_per_rpc_report` that tells the server to update per-query metrics.
+2. A `TestLoadReport` type `orca_oob_report` that tells the server to update OOB metrics.
+
+Server features:
+* [UnaryCall][]
+* [Backend metrics report][]
+
+#### Per-query metrics reporting
```

Comment:
This is written as two different tests. Write it as two separate tests? `orca_per_rpc` and `orca_oob`? Don't follow the soak test style; it looks like it has incorrect formatting and it is a very different test than normal.

The `test_backend_metrics_load_balancer` discussion can just be on one of them and then referenced in the second.

Reply:
oh, do you suggest to use two separate test cases at the interop-testing client test cases?